src/components/inputs/z-input/index.tsx

```diff
@@ -258,8 +259,31 @@ export class ZInput {
     );
   }
 
+  renderEye(status: string) {
+    return (
+      <z-icon
+        name={status}
+        onClick={() => {
+          this.passwordHidden = !this.passwordHidden;
+        }}
+      />
+    );
+  }
+
+  renderShowHidePassword() {
+
+    if (this.disabled) this.hostElement.style.pointerEvents = 'none';
+
+    return this.passwordHidden ? this.renderEye('hide-password') : this.renderEye('show-password');
+  }
+
   renderResetIcon() {
-    if (!this.value || this.disabled || this.readonly) return;
+    if (
+      !this.value ||
+      this.disabled ||
+      this.readonly 
+    )
+      return;
```

Comment:
check `return` undefined for Edge Compat

Reply:
I didn't change it, renderResetIcon func already existed ..and it's working fine on Edge